docs/glossary.md

```diff
@@ -0,0 +1,242 @@
+
+An equip offer looks like this:
+
+```js
+const equipProposal = {
+    // the following assets are sent from your wallet to Zoe
+    give: {
+        // the character SFT you wish to equip to, it's temporarily sent to Zoe as proof of ownership and will be returned
+        CharacterKey2: {
+            brand: characterBrand,
+            value: makeCopyBag([[characterToEquipTo, 1n]]),
+        },
+        // the item SFT you wish to equip
+        Item: {
+            brand: itemBrand,
+            value: makeCopyBag([[itemToEquip, 1n]]),
+        },
+    },
+    // the following assets must be received or the give will be returned
+    want: {
+        // the character must be returned
+        CharacterKey1: { brand: characterBrand, value: makeCopyBag([[characterToEquipTo, 1n]]) },
+    }
+};
+
+```
+
+#### Unequip
+The unequip action allows the owner of a [Character](#character-sft) to transfer an item from the Character's Inventory to their own wallet. Only one Item can be unequipped at a time and the only requirement is ownership of the Character. A user may want to unequip an item for the following reasons:
+1. To change the properties of a Character (look and stats)
+2. To manually equip a different item of the same category (consider using [swap](#swap) in this situation)
+3. To sell the item in the marketplace
+   
+An unequip proposal looks like this:
+```js
+const unequipProposal = {
+    // the following assets are sent from your wallet to Zoe
+    give: {
+        // the character SFT you wish to equip to, it's temporarily sent to Zoe as proof of ownership and will be returned
+        CharacterKey1: { brand: charBrand, value: makeCopyBag([[characterToEquipTo, 1n]]) },
+    }
+    // the following assets must be received or the give will be returned
+    want: {
+        // the character must be returned
+        CharacterKey2: {
+            brand: charBrand,
+            value: makeCopyBag([[characterToEquipTo, 1n]]),
+        },
+        // the item SFT you wish to unequip
+        Item: {
+            brand: itemBrand,
+            value: makeCopyBag([[wantedItem, 1n]]),
+        },
+    },
+};
+```
+
+#### Swap
+The swap action allows the owner of a [Character](#character-sft) to swap an [Inventory](#inventory) item for one from their wallet. Since only one Item per [category](#item-category) can be equipped at a time, replacing an equipped item would require two separate transaction (first unequip, then equip new item). The swap action combines it into a single transaction and allows users to replace an equipped item for anotherone of the same category. 
```

Comment:
```suggestion
The swap action allows the owner of a [Character](#character-sft) to swap an [Inventory](#inventory) item for one from their wallet. Since only one Item per [category](#item-category) can be equipped at a time, replacing an equipped item would require two separate transaction (first unequip, then equip new item). The swap action combines it into a single transaction and allows users to replace an equipped item for another one of the same category. 
```